fpdf/fpdf.py

```diff
@@ -2135,12 +2136,27 @@ def _perform_page_break(self):
         if ws > 0:
             self.ws = 0
             self._out("0 Tw")
-        self.add_page(same=True)
+        self._break_or_add_page()
```

Comment:
I'd prefer not to add introduce any new method here:
as their code is not supposed to be called elsewhere,
it could simply be inlined.